src/test/java/htsjdk/samtools/cram/structure/CramCompressionRecordTest.java

```diff
@@ -12,19 +13,24 @@
  * Created by vadim on 28/09/2015.
  */
 public class CramCompressionRecordTest extends HtsjdkTest {
-    @Test
-    public void test_getAlignmentEnd() {
+    @DataProvider(name = "tf")
```

Comment:
Can we give this a name that indicates what its intended to toggle (i.e. APDelta).